x-pack/solutions/security/test/security_solution_api_integration/kibana.jsonc

```
@@ -0,0 +1,8 @@
+{
+  "type": "test-helper",
+  "id": "@kbn/security_solution_api_integration",
```

This package is mainly FTR tests, with the sub-package above containing the test helpers. Can we rename this and/or add a new package type to reflect this fact?

```suggestion
  "type": "functional-tests",
  "id": "@kbn/security-solution-api-ftr-tests",
```

---

Or: I see other packages follow the `@kbn/test-suites-` prefix; so maybe `@kbn/test-suites-security-solution-apis` if that's your preference?